mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -169,6 +191,8 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 		METHOD method_4339 setPackAlignment ()V
 		METHOD method_4340 setUnpackAlignment ()V
 	CLASS class_1013 GLFormat
```

Comment:
This is a weird name, shouldn't it be something like `ColorFormat` or `PixelFormat`?

Reply:
I'll address this in a later PR